### DISCUSSION TITLE
ChannelFollows sha256Hash

### DISCUSSION THREAD

**Discussion by rdavydov** [RakambdaOrg/ChannelPointsMiner#440] [Q&A] [answered]
Hi @Rakambda ,

How can one get the sha256Hash of 'ChannelFollows' persistedQuery?

My logs show that "4b9cb31b54b9213e5760f2f6e9e935ad09924cac2f78aac51f8a64d85f028ed0" is outdated, as it throws the following error:

```
[post_gql_request]: Data: {'operationName': 'ChannelFollows', 'variables': {'limit': 100, 'order': 'ASC', 'cursor': ''}, 'extensions': {'persistedQuery': {'version': 1, 'sha256Hash': '4b9cb31b54b9213e5760f2f6e9e935ad09924cac2f78aac51f8a64d85f028ed0'}}}, Status code: 200, Content: {"errors":[{"message":"PersistedQueryNotFound"}],"extensions":{"durationMilliseconds":2,"operationName":"ChannelFollows","requestID":"XXX"}}
```

I've tried sniffing the android app (with ssl-pinning hacked of course), but got no such query.

Thanks in advance!

**Comment by Rakambda** [comment, marked as answer]
Navigate to https://www.twitch.tv/directory/following/channels

![image](https://user-images.githubusercontent.com/1688389/221243078-48345b7e-92e6-4b33-a87c-e95284cf19c5.png)
